resource/js/util/PreProcessor/Linker.js

```diff
@@ -1,10 +1,23 @@
+var path = require('path');
 
 export default class Linker {
   process(markdown) {
 
     return markdown
       //.replace(/\s(https?:\/\/[\S]+)/g, ' <a href="$1">$1</a>') // リンク
       .replace(/\s<((\/[^>]+?){2,})>/g, ' <a href="$1">$1</a>') // ページ間リンク: <> でかこまれてて / から始まり、 / が2個以上
+      // Pukiwiki Like Linker
+      // see: https://regex101.com/r/k2dwz3/3
+      .replace(/\[\[(([^(\]\])]+)>)?(.+?)\]\]/g, function(all, group1, group2, group3) {
+        // create url
+        // http(s) から始まっていればそのまま利用、そうでなければ window.location.pathname と連結
+        var url = (group3.match(/^https?:\/\//)) ? group3 : path.join(window.location.pathname, group3);
```

Comment:
It may not work on a list page and a timeline view.
The page would be rendered not only on a single page, but also on any list page. Therefore, join with the current path could make a wrong link.

example:

- if the page is`/foo/bar/baz` and has `./hoge`
    - it should link to `/foo/bar/hoge`
    - it works on a single page
- if user access to `/foo/` (a list page)
    - and show timeline view,
    - then, the link may point to `/foo/hoge`